src/utils/json-data.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs-extra');
+const path = require('path');
+const reserved = require('reserved-words');
+const shared = require('./shared');
+const glob = require('glob');
+const BUILD_TIME_BINDING = '@buildtime:';
+// Matches {{ stache.jsonData. }} with any number of spaces between the '{{' and 'stache.',
+// any keys following the 'jsonData.', and the closing '}}'
+const angularBindingRegExp = new RegExp(/\{\{\s*stache.jsonData.*?\}\}/g);
+const buildTimeBindingRegExp = new RegExp(`\\{\\{\\s*${BUILD_TIME_BINDING}stache.jsonData.*?\\}\\}`, 'g');
+
+let _globalData;
+
+const getGlobalData = () => {
```

Comment:
I know how much you like semantics :), but I'm having a tough time with the use of "global" here. We're recommending that folks include a "global" JSON file to handle SPA configuration (see: `stache.jsonData.global`, https://github.com/blackbaud/skyux-template-stache/blob/master/src/stache/data/global.json), so I feel like we'd be creating undue confusion by using "global" here too (I'm thinking specifically of a non-UserEd developer who wants to contribute to the plugin). 

This plugin is consolidating all JSON files into a single JavaScript Object, not just the "global" JSON data (which is why I chose `dataObject` originally). You don't have to use that name, but something that denotes "JSON file being parsed as a JavaScript Object" would be more appropriate.

If you're sick and tired of wording, maybe @blackbaud-brandonhare can pull out his fancy dictionary.